packages/framer-motion/src/render/dom/scroll/index.ts

```diff
@@ -86,6 +86,7 @@ function needsElementTracking(options?: ScrollOptions) {
 
 function scrollFunction(onScroll: OnScroll, options: ScrollOptions) {
     if (isOnScrollWithInfo(onScroll) || needsElementTracking(options)) {
+        console.log(isOnScrollWithInfo(onScroll), needsElementTracking(options))
```

Comment:
Just a reminder that this should be removed before release! Apologies if you were already aware of this :3